world.js

```diff
@@ -106,36 +106,41 @@ world.connectRoom = async (roomName, worldURL) => {
       channelConnection.setMicrophoneMediaStream(networkMediaStream);
     }
     
-    if (live) {
-      interval = setInterval(() => {
-        const name = loginManager.getUsername();
-        const avatarSpec = loginManager.getAvatar();
-        const avatarUrl = avatarSpec && avatarSpec.url;
-        const avatarExt = avatarSpec && avatarSpec.ext;
-        const address = loginManager.getAddress();
-        const aux = rigManager.localRig?.aux.getPose();
-        channelConnection.send(JSON.stringify({
-          method: 'status',
-          data: {
-            peerId: channelConnection.connectionId,
-            status: {
-              name,
-              avatarUrl,
-              avatarExt,
-              address,
-              aux,
-            },
-          },
-        }));
-        const pose = rigManager.getLocalAvatarPose();
-        channelConnection.send(JSON.stringify({
-          method: 'pose',
-          data: {
-            pose,
+
+    interval = setInterval(() => {
```

Comment:
Why is being live no longer a requirement?

Reply:
live wasn't defined in that scope

Reply:
Good catch.